admin/server.go

```diff
@@ -32,18 +38,19 @@ func StartNKNServer(account *nkn.Account, identifier string, clientConfig *nkn.C
 
 		isAcceptAddr := util.MatchRegex(persistConf.GetAcceptAddrs(), msg.Src)
 		isAdminAddr := util.MatchRegex(persistConf.GetAdminAddrs(), msg.Src)
+		isNetworkMemberAddr := util.MatchRegex(networkMemberIAccept, msg.Src)
 
 		if !isAdminAddr && tokenStore.IsValid(req.Token) {
 			isAdminAddr = true
 		}
 
-		if !isAcceptAddr && !isAdminAddr {
-			log.Println("Ignore authorized message from", msg.Src)
+		if !isAcceptAddr && !isAdminAddr && !isNetworkMemberAddr {
+			log.Println("Ignore unauthorized message from", msg.Src)
 			continue
 		}
 
 		var perm permission
-		if isAcceptAddr {
+		if isAcceptAddr || isNetworkMemberAddr {
```

Comment:
We should separate the permissions of accept addr and member addr

Reply:
Yes, modified.